### DISCUSSION TITLE
[Feature] Multiple album admins

### DISCUSSION THREAD

**Discussion by matrixes** [immich-app/immich#12873] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have a request stemming from people using Immich, where they would like to be able to remove photos from a shared album without having to actively contact the album owner or resort to commenting on the photo as a request to get it removed.

The admin of an album can remove any photo in an album, but there's only one admin. Making it possible to edit what permission an invited user has (per album), would be useful.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
We're tracking this in general in #12614
